webpack/ForemanInventoryUpload/ForemanInventoryConstants.js

```diff
@@ -0,0 +1,4 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+
+export const INVENTORY_PAGE_TITLE = __('Red Hat Inventory Uploads');
+export const INVENTORY_BASE_URL = '/foreman_inventory_upload/';
```

Comment:
We can't assume foreman lives in root url (`https://example.com`). You can probably use foreman's base function.